pkg/scm/scm.go

```diff
@@ -45,8 +45,7 @@ func DownloaderForSource(fs util.FileSystem, s string, forceCopy bool) (build.Do
 		return &file.File{FileSystem: fs}, s, nil
 	}
 
-	// If the source is valid  Git protocol (file://, ssh://, git://, git@, etc..) use Git
```

Comment:
@rhcarvalho that is true(that the comment would be better served elsewhere and will get overlooked during updates here), so please move this comment into ValidCloneSpec.